tests/scripts/analyze_outcomes.py

```diff
@@ -210,9 +213,11 @@ def run(self, results: Results, outcomes: Outcomes) -> None:
             elif hit and ignored:
                 # Test Case should be removed from the allow list.
```

Comment:
Absolute Nit, but the comment is still wrong - could probably just be removed.

Reply:
Oops, I meant to rewrite both the message and the comment, and I forgot the comment. Done (I amended the last commit).